README.md

```diff
@@ -108,8 +108,7 @@ class CustomStyle: DayStyle {
 
     override func apply() {
         super.apply()
-        ManeuverView.appearance().backgroundColor = .darkGray
-        RouteTableViewHeaderView.appearance().backgroundColor = .darkGray
+        BottomBannerView.appearance().backgroundColor = .orange
```

Comment:
Would it make sense to add a link to DayStyle.swift#master to showcase all styling capabilities w/o having to update this readme constantly?

Reply:
Yes, that's a great idea.

Reply:
Added link in https://github.com/mapbox/mapbox-navigation-ios/pull/923/commits/259a294a5306eda3ba38da4e11fc64c50b006482. Let's keep this one line though so the user gets a feeling for how the API looks.